tahoe_sites/helpers.py

```diff
@@ -0,0 +1,106 @@
+"""
+Package helper methods goes here
+"""
+from django.contrib.auth import get_user_model
+from organizations.models import Organization
+
+from tahoe_sites import zd_helpers
+from tahoe_sites.models import TahoeSiteUUID, UserOrganizationMapping
+
+
+def get_organization_by_uuid(organization_uuid):
+    """
+    Get an organization object from it's uuid
+
+    :param organization_uuid: uuid to filter on
+    :return: organization of the given uuid
+    """
+    if zd_helpers.should_site_use_org_models():
+        return Organization.objects.get(edx_uuid=organization_uuid)
+    return TahoeSiteUUID.objects.get(site_uuid=organization_uuid).organization
+
+
+def get_uuid_by_organization(organization):
+    """
+    Get the uuid when from it's related organization
+
+    :param organization: organization to filter on
+    :return: uuid of the given organization
+    """
+    if zd_helpers.should_site_use_org_models():
+        return organization.edx_uuid
+    return TahoeSiteUUID.objects.get(organization=organization).site_uuid
+
+
+def get_organizations_for_user(user, with_inactive_users=False, without_admins=False):
+    """
+    Return organizations related to the given user. By default, the list will ignore
+    organizations where the user is inactive
+
+    :param user: user to filter on
+    :param with_inactive_users: include organizations where the user is inactive (default = False)
+    :param without_admins: exclude organizations where the user is set as admin (default = False)
+    :return: Organization objects related to the given user
+    """
+    if with_inactive_users:
+        extra_params = {}
+    else:
+        extra_params = {'is_active': True}
+    if without_admins:
+        extra_params['is_admin'] = False
+
+    return Organization.objects.filter(
+        pk__in=UserOrganizationMapping.objects.filter(user=user, **extra_params).values('organization')
```

Comment:
Explicit is better than implicit.

```suggestion
        pk__in=UserOrganizationMapping.objects.filter(user=user, **extra_params).values('organization_id')
```

Reply:
done